src/main/kotlin/io/S5p.kt

```diff
@@ -58,10 +63,31 @@ object S5p {
         model.Track(
             id = index,
             name = track.name ?: "Track ${index + 1}",
-            notes = parseNotes(track)
+            notes = parseNotes(track),
+            pitch = parsePitch(track)
         ).validateNotes()
     }
 
+    private fun parsePitch(track: Track): Pitch? {
+        val pitchDelta = track.parameters?.pitchDelta ?: return Pitch(emptyList(), isAbsolute = false)
+        val convertedPoints = pitchDelta.asSequence()
+            .withIndex()
+            .groupBy { it.index / 2 }
+            .map { it.value }
+            .map { it.map { indexedValue -> indexedValue.value } }
+            .mapNotNull {
+                val rawTick = it.getOrNull(0) ?: return@mapNotNull null
+                val centValue = it.getOrNull(1) ?: return@mapNotNull null
+
+                val tick = rawTick * 3.75
```

Comment:
`3.75` is actually `interval / TICK_RATE`, so we should avoid using such a magic number